src/lib/utils/config/src/statics.rs

```diff
@@ -1,21 +1,22 @@
-//! # Statics module.
-//!
-//! Contains the static global configuration and its related functions.
-
 use crate::server_config::ServerConfig;
+use ferrumc_core::identity::player_identity::PlayerIdentity;
 use ferrumc_general_purpose::paths::get_root_path;
 use lazy_static::lazy_static;
 use std::fs::File;
 use std::io::{Read, Write};
 use std::process::exit;
-use tracing::{error, info};
+use std::sync::Mutex;
+use tracing::{debug, error, info};
 
 /// The default server configuration that is stored in memory.
 pub(crate) const DEFAULT_CONFIG: &str = include_str!("../../../../../.etc/example-config.toml");
 
 lazy_static! {
     /// The server configuration that is stored in memory.
     static ref CONFIG: ServerConfig = create_config();
+    /// The whitelist of players, wrapped in a Mutex for thread-safe access
+    /// assuming commands can come from any thread in future
+    static ref WHITELIST: Mutex<Vec<PlayerIdentity>> = Mutex::new(create_whitelist()); //should this be an rwlock instead? does it even need to be in statics?
```

Comment:
Yeah should be a rwlock, but ideally you would have this as a oncelock or something, since you really only need to modify it once when the program is started

Reply:
Actually just thought about it some more, you could just have a DashSet of usernames to check, that way you don't have to deal with any synchronization yourself and you'd have O(1) lookup. This would make using json unfeasible but i've already suggested you don't do that.